Sources/DictionarySpecs/dictionaryManagerSpec.swift

```diff
@@ -0,0 +1,14 @@
+import Dictionary
+import Swiftest
+
+class DictionaryManagerSpec: Swiftest.Spec {
+	let spec = describe("dictionary logic") {
+		describe("#getRandomWord") {
+			it("returns a random word from the dictionary") {
+				let dictionaryManager = DictionaryManager()
+				let word = dictionaryManager.getRandomWord()
+				expect(dictionaryManager.gameWords).to.contain(word)
+			}
```

Comment:
I am not sure what you are referring to. I am not testing `getRandomIndex` because it is a private method and because it's utility is tested when I call the `getRandomWord` function. If there is something else to test in `getRandomWord`, I need a hint please